### ISSUE TITLE
Badge/Tag - Update colors for better accessibility contrast

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#5673]
**Describe the bug**
Make some style changes to badge and tag colors. 

**Fixes**
https://www.figma.com/file/gHmapzCfCqqPRv5RnFfGcE/?node-id=0%3A1
- [ ] Change colors in all 3 variants (new)
- [ ] Update the badge and tag examples to match
- [ ] add status colors to https://main-enterprise.demo.design.infor.com/components/colors/example-index.htm
- [ ] Make changes in the web component too
- [ ] Make sure tokens are in sync

**Additional context**
https://jira.infor.com/browse/UXD-1865

**Comment by janahintal** [comment]
QA Failed. 
Light UI Expected color for Status 04 is #1F9254
Result: #1C7F49

Figma: https://www.figma.com/file/gHmapzCfCqqPRv5RnFfGcE/Q3-Accessibility-Tests
![image](https://user-images.githubusercontent.com/45757016/149758568-139dc5a5-1492-465a-a2f4-20e55a069d5b.png)

**Comment by ericangeles** [comment]
@janahintal can you please provide the steps for this test? Thanks.

**Comment by ericangeles** [comment]
@tmcconechy @kevinwhitedesign can we confirm the color that we should use for the New Light? In the example page, we used emerald09 (#1C7F49) but in the Figma file it's #1F9254 (emerald08).

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.

**Comment by tmcconechy** [comment]
I will reverse some of these changes for now due to https://github.com/infor-design/enterprise/issues/6114
The reason being the size and font changes look abrupt and out of place and want to avoid complaints and breaking.

The biggest problem being the size change on the fields that will look out of place on forms with different fields
![Screen Shot 2022-02-10 at 2 38 29 PM](https://user-images.githubusercontent.com/814283/153483829-4b1d33d0-339e-4f11-84fc-a8b395e02c17.png)

And the tag size changes in general can cause disruption. So will kick this back for now

Approved with @inforandy

**Comment by kevinwhitedesign** [comment]
I believe reverting from this change will end in failing accessibility latest WCAG guidelines. Correct me if I'm wrong @elizabethhartley

**Comment by tmcconechy** [comment]
So according to a scan on https://4580-enterprise.demo.design.infor.com/components/tag/example-index.html only one thing fails...

New Light:
- only the green tag fails -> can easier be fixed by using slate 10 instead of white

So rather easy to just change one color then change the size of the tags and multiselect and make the multiselect look different on forms. Just has too much impact for this change but will keep it in the backlog for later.

**Comment by kevinwhitedesign** [comment]
@tmcconechy oh right, we decided to make them all bold for consistency. @elizabethhartley reminded me

**Comment by tmcconechy** [comment]
Yeah i think in this case we should come back to this one and just have:

- check the 3 themes
- only change the background colors up and down
- change the text color between white or slate 100 depending which bg color
- not change any sizes

Also wondering about yellow or orange alerts - which is most often done
